formats/detailed_guide/frontend/examples/detailed_guide.json

```diff
@@ -0,0 +1,71 @@
+    "change_note": "The second entry in the table Examples of salary sacrifice has been amended to correct the explanation of how much of the salary is subject to tax and National Insurance contributions.",
```

Comment:
The live example, https://www.gov.uk/guidance/salary-sacrifice-and-the-effects-on-paye shows a complete history of changes. We should represent that here.